app/src/main/java/org/gdg/frisbee/android/app/App.java

```diff
@@ -127,9 +131,46 @@ public void onCreate() {
         mLocationFinder = new GingerbreadLastLocationFinder(this);
         mLocationFinder.setChangedLocationListener(this);
         updateLastLocation();
+        
+        //Init TaggedEventSeries
+        mTaggedEventSeriesList = new ArrayList<>();
+        addTaggedEventSeriesIfDateFits(new TaggedEventSeries("wtm",
+                R.drawable.drw_ic_wtm,

```

Comment:
while wtm is the tag, I think women techmakers should be abbreviates as wt, but that is not an issue here.